uefi/src/proto/ata/mod.rs

```diff
@@ -0,0 +1,322 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+//! ATA Protocols.
+
+use crate::helpers::{AlignedBuffer, AlignmentError};
+use core::alloc::LayoutError;
+use core::marker::PhantomData;
+use core::ptr;
+use core::time::Duration;
+use uefi_raw::protocol::ata::{
+    AtaCommandBlock, AtaPassThruCommandPacket, AtaPassThruLength, AtaStatusBlock,
+};
+
+pub mod pass_thru;
+
+/// Represents the protocol for ATA Pass Thru command handling.
+///
+/// This type defines the protocols supported for passing commands through the ATA controller.
+pub type AtaPassThruCommandProtocol = uefi_raw::protocol::ata::AtaPassThruCommandProtocol;
```

Comment:
wait. Isn't this equivalent to `pub use uefi_raw::protocol::ata::AtaPassThruCommandProtocol`?